symbols/arm9.yml

```diff
@@ -10134,6 +10139,94 @@ arm9:
       description: |-
         r0: Sound envelope pointer
         return: Current volume
+    - name: SoundLfoBankReset
+      address:
+        EU: 0x2075434
+        NA: 0x207509C
+      description: "r0: LFO bank pointer"
+    - name: SoundLfoBankSet
+      address:
+        EU: 0x207544C
+        NA: 0x20750B4
+      description: |-
+        r0: LFO bank pointer
+        r1: LFO settings pointer
+        r2: Envelope level
+    - name: SoundLfoBankSetConstEnvelopes
+      address:
+        EU: 0x2075644
+        NA: 0x20752AC
+      description: |-
+        r0: LFO bank pointer
+        r1: Level
+    - name: SoundLfoBankTick
+      address:
+        EU: 0x2075690
+        NA: 0x20752F8
+      description: "r0: LFO bank pointer"
```

Comment:
The prototype says this returns a `uint16_t`?

Reply:
Added.